cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/YamlConfig.kt

```diff
@@ -37,7 +37,9 @@ class YamlConfig(
         val limitBreak: Boolean = false,
         val projectId: String = getDefaultProjectId(),
         val devices: List<Devices> = listOf(),
-        var testShardChunks: Set<Set<String>> = emptySet()) {
+        var testShardChunks: Set<Set<String>> = emptySet(),
```

Comment:
I don't see `testShardChunks` in the yaml in this PR. 
What would the yaml look like for that?

Reply:
`testShardChunks` is a calculated properly based on `testShards` in the `flank.yml` It's not set in the yaml file directly.

Reply:
ah, I see. Also, not new in this PR. :) 

Having that as a property, could be confusing to someone looking at YamlConfig.kt for what types of things they can put into the .yaml file.

Reply:
Yeah, it may be time to refactor the configuration into different classes.

YamlConfig
IosConfig
AndroidConfig

There's a lot happening in the class now.

Reply:
I added a ticket to track this idea:
https://github.com/TestArmada/flank/issues/224